lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -21,9 +21,49 @@
             <layout.heading local-class='Heading'>
                 <div local-class='Title'>{{t 'registries.drafts.draft.form.new_registration'}}</div>
             </layout.heading>
-            <layout.left local-class='Left'>
-                {{!-- Sidenav: page headings, validations --}}
-            </layout.left>
+            
+            <layout.leftNav as |leftNav|>
+                {{#each draftManager.pageManagers as |pageManager index|}}
+                    <PageLinkManager
+                        @pageManager={{pageManager}}
+                        @pageIndex={{index}}
+                        @currentPage={{draftManager.currentPage}}
+                        as |pageLinkManager|
+                    >
+                        <leftNav.link
+                            data-analytics-name='{{pageLinkManager.IndexWithSlug}}'
+                            class={{local-class (compute (action this.pageStateToClass) pageLinkManager.pageState)}}
```

Comment:
the above suggestions could help us make this a simple `computed` prop.

Reply:
Indeed, creating a `PageLink` component that did this in a computed would be better.